readme.md

```diff
@@ -211,6 +213,14 @@ To mark one or more items as favorite, use the `--star`/`-s` option followed by
 $ tb -s 1 2 3
 ```
 
+### Copy to the clipboard
+
+Copy a task/note description to the system clipboard.
+
+```
+$ tb -g 2
+```
+
```

Comment:
It would be nice to elaborate a bit more on the flight manual, for example: 

### Copy Item Description

To copy to the your system's clipboard the description of one or more items, use the `--copy`/`-y` option followed by the ids of the target items. Note that the option will also include the newline character as a separator to each pair of adjacent copied descriptions, thus resulting in a clear and readable stack of sentences on paste.

```
$ tb -y 1 2 3
```